pkg/snowflake/stage.go

```diff
@@ -190,3 +193,76 @@ func (sb *StageBuilder) Describe() string {
 func (sb *StageBuilder) Show() string {
 	return fmt.Sprintf(`SHOW STAGES LIKE '%v' IN DATABASE "%v"`, sb.name, sb.db)
 }
+
+type stage struct {
+	Name               *string `db:"name"`
+	DatabaseName       *string `db:"database_name"`
+	SchemaName         *string `db:"schema_name"`
+	Comment            *string `db:"comment"`
+	StorageIntegration *string `db:"storage_integration"`
+}
+
+func ScanStageShow(row *sqlx.Row) (*stage, error) {
+	r := &stage{}
+	e := row.StructScan(r)
+	return r, e
+}
+
+type descStageResult struct {
+	Url              string
+	AwsExternalID    string
+	SnowflakeIamUser string
+	FileFormat       string
+	CopyOptions      string
+}
+
+type descStageRow struct {
+	parentProperty  string `db:"parent_property"`
+	property        string `db:"property"`
+	propertyType    string `db:"property_type"`
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`propertyType` is unused (structcheck)